cfme/tests/automate/test_instance.py

```diff
@@ -280,3 +281,78 @@ def test_check_system_request_calls_depr_conf_mgmt(appliance, copy_instance):
         request=copy_instance.name
     )
     result.validate_logs()
+
+
+@pytest.fixture(scope="module")
+def copy_quota_instance(domain):
+    """Copy the default instance 'quota' to custom domain"""
+    miq = domain.appliance.collections.domains.instantiate("ManageIQ")
+
+    original_instance = (
+        miq.namespaces.instantiate("System")
+        .namespaces.instantiate("CommonMethods")
+        .classes.instantiate("QuotaStateMachine")
+        .instances.instantiate("quota")
+    )
+    original_instance.copy_to(domain=domain)
+
+    instance = (
+        domain.namespaces.instantiate("System")
+        .namespaces.instantiate("CommonMethods")
+        .classes.instantiate("QuotaStateMachine")
+        .instances.instantiate("quota")
+    )
+    return instance
+
+
+@pytest.mark.tier(2)
+@pytest.mark.parametrize(
+    ("entity", "search"),
+    [
+        (
+            'group',
+            'Getting Quota Values for Model',
+        ),
+        (
+            'tenant',
+            'Getting Tenant Quota Values for',
+        ),
+    ],
+    ids=["group", "tenant"],
+)
+def test_quota_source_value(request, entity, search, copy_quota_instance, generic_catalog_item):
+    """
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseposneg: positive
+        startsin: 5.10
+        casecomponent: Automate
+
+    Bugzilla:
+        1319910
+    """
+    # Changing quota source value
+    copy_quota_instance.update({"fields": {'quota_source_type': {'value': entity}}})
+
+    # Setting cpu quota for 'My Company' tenant
+    root_tenant = copy_quota_instance.appliance.collections.tenants.get_root_tenant()
+    root_tenant.set_quota(**{"cpu_cb": True, "cpu": 3})
+    request.addfinalizer(lambda: root_tenant.set_quota(**{"cpu_cb": False}))
+
+    result = LogValidator(
+        "/var/www/miq/vmdb/log/automation.log", matched_patterns=[".*{}.*".format(search)]
+    )
+    result.fix_before_start()
+    service_catalogs = ServiceCatalogs(
+        copy_quota_instance.appliance, catalog=generic_catalog_item.catalog,
+        name=generic_catalog_item.name
+    )
+    request_description = 'Provisioning Service [{name}] from [{name}]'.format(
+        name=service_catalogs.name)
+    provision_request = copy_quota_instance.appliance.collections.requests.instantiate(
+        description=request_description)
+    service_catalogs.order()
+    provision_request.wait_for_request(method='ui')
+    request.addfinalizer(lambda: provision_request.remove_request(method="rest"))
+    result.validate_logs()
```

Comment:
**QUESTION** I am trying to understand how are you making sure logs are valid? Does `validate_logs()` returns a boolean that you can assert? I checked https://github.com/ManageIQ/integration_tests/blob/32faa0a0fba7dde3a611bc45b3ef653285896457/cfme/utils/log_validator.py#L55 and don't see a return value. 

I might be missing something but can you elaborate how/where you `assert` match?